src/mono/mono/mini/mini-runtime.c

```diff
@@ -118,6 +118,7 @@ const char *mono_build_date;
 gboolean mono_do_signal_chaining;
 gboolean mono_do_crash_chaining;
 int mini_verbose = 0;
+gboolean term_signaled = FALSE;
```

Comment:
nit: `mono_term_signaled`.
nit: `volatile` ?

Reply:
Changed the variable name. As long as it's not spinning on that value I assume it will reload on each reference?